.github/workflows/ci.yml

```diff
@@ -12,30 +12,34 @@ jobs:
     strategy:
       matrix:
         mcu:
-          - stm32f301
-          - stm32f318
+          - stm32f301x6
+          - stm32f318x8
+          - stm32f302x6
+          # - stm32f302x8
```

Comment:
Why are these currently commented out?

Reply:
Devices only differ in memory size are commented out as they only check the literally same code.

Reply:
Hm. This does make sense. Good catch :+1: Could you at a comment, so this is obvious, when editing the file, so this will not be accidentally deleted? :)